src/proto/BitcoinV2.proto

```diff
@@ -0,0 +1,249 @@
+syntax = "proto3";
+
+package TW.BitcoinV2.Proto;
+option java_package = "wallet.core.jni.proto";
+
+import "Common.proto";
+import "Utxo.proto";
+
+// TODO: Update numbers/sorting.
+enum Error {
+    OK = 0;
+    Error_invalid_private_key = 1;
+    Error_invalid_public_key = 2;
+    Error_invalid_sighash = 3;
+    Error_invalid_sighash_type = 16;
+    Error_invalid_witness_pubkey_hash = 4;
+    Error_brc20_invalid_ticker = 5;
+    Error_invalid_ecdsa_signature = 11;
+    Error_invalid_schnorr_signature = 14;
+    Error_invalid_control_block = 12;
+    Error_invalid_pubkey_hash = 13;
+    Error_invalid_taproot_root = 15;
+    Error_unmatched_input_signature_count = 17;
+    Error_missing_input_builder = 18;
+    Error_missing_output_builder = 20;
+    Error_missing_recipient = 19;
+    Error_unsupported_address_recipient = 21;
+    Error_bad_address_recipient = 22;
+}
+
+message SigningInput {
+    // The protocol version, is currently expected to be 1 or 2 (BIP68)
+    int32 version = 1;
+
+    // Only required if the `sign` method is called.
+    bytes private_key = 2;
+
+    // Block height or timestamp indicating at what point transactions can be
+    // included in a block.
+    Utxo.Proto.LockTime lock_time = 3;
+
+    repeated Input inputs = 5;
+
+    repeated Output outputs = 6;
+
+    Utxo.Proto.InputSelector input_selector = 7;
+
+    uint64 sat_vb = 8;
+
+    // The change output to be added (return to sender).
+    // The `amount` can be left at 0.
+    Output change_output = 9;
+
+    // Explicility disable change output creation.
+    bool disable_change_output = 10;
```

Comment:
We can make `change_output ` optional

Reply:
See https://github.com/trustwallet/wallet-core/pull/3382#discussion_r1302011295